### ISSUE TITLE
TFT35 throws M14 E unknown command

### ISSUE THREAD

**Issue by Blisk** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2960]
### Description

When I print something I get error unknown command M14 E
### Steps to reproduce

Start printing over octoprint and get this error

Print withouth errors

### Hardware Variant

TFT35 V3
### Additional Information
* Include a ZIP file containing your `Configuration.h` or use [Pastebin](https://pastebin.com/) and paste a link in this issue.
* Provide pictures or links to videos that clearly demonstrate the issue.
![20241016_091502](https://github.com/user-attachments/assets/28c6dda8-d702-459b-b09e-705167efbbc2)
![20241016_092032](https://github.com/user-attachments/assets/b0e599cf-bc93-4deb-8cd1-75e25a1d079f)

**Comment by rondlh** [comment]
This is an error thrown by the motherboard host software, the display just reports it. 
I'm not sure what host software you are running, but Marlin doesn't understand M14, so it would report the error you got.
(https://marlinfw.org/docs/gcode/M010-M011.html)

Make sure to select the correct gcode dialect when slicing.

**Comment by Blisk** [comment]
it is also Marlin 2.1.X on SKR mini 3.0

**Comment by radek8** [comment]
This error is usually caused by electromagnetic interference on the serial port

**Comment by Blisk** [comment]
> This error is usually caused by electromagnetic interference on the serial port. SKR mini V3 boards suffer a lot from this

I have now shielded sreial cable

**Comment by radek8** [comment]
It doesn't always help. It looks like a bad design of the board, maybe the interference from the stepper motor controller is oscillating to the serial port. Put the cables as far away from the stepper motor controller as possible.

**Comment by Blisk** [comment]
> It doesn't always help. It looks like a bad design of the board, maybe the interference from the stepper motor controller is oscillating to the serial port. Put the cables as far away from the stepper motor controller as possible.

OK I will do that also.

**Comment by radek8** [comment]
A lot of people here complain about this problem and 90% of them have one thing in common - SKR mini V3.0

**Comment by rondlh** [comment]
@Blisk Do you always get the "M14 E" error, or did it just happen once? If it only happened once then, like @radek8 mentioned, it could be a serial communication reliability issue. In that case lower your serial baud rate to 115200 baud, shorten your serial cable as much as possible and keep the serial cable away from the stepper cables.

**Comment by Blisk** [comment]
> @Blisk Do you always get the "M14 E" error, or did it just happen once? If it only happened once then, like @radek8 mentioned, it could be a serial communication reliability issue. In that case lower your serial baud rate to 115200 baud, shorten your serial cable as much as possible and keep the serial cable away from the stepper cables.

yes that error happend many times and I have shielded serial cable.

**Comment by radek8** [comment]
Is the error code still the same, or is it changing and is different each time? 
(text in quotation marks)

**Comment by Blisk** [comment]
code is mostly the same

**Comment by kisslorand** [comment]
@Blisk 

Test the FW from [my repo](https://github.com/kisslorand/BTT-TFT-FW). Some monkey loitering around (@rondlh) might tell you to beware because that FW might burn your house, empty your bank accounts, break your printer or any other stupidity he comes up with but you should be smart enough to think for yourself. 
Other people did the switch and never looked back.

**Comment by Blisk** [comment]
Hi

I checked your FW but you only have TFT28 firmware i use TFT35 display.

What I need to do here?

**Comment by digant73** [comment]
@Blisk that unknown command messages are probably fixed by PR #2970. Increase the value for new param `tx_delay` (you can change it also at runtime from the Feature menu) until the issue is no more present.
